modules/formulize/language/french/modinfo.php

```diff
@@ -66,3 +66,8 @@
 
 define("_MI_formulize_USEOLDCUSTOMBUTTONEFFECTWRITING", "Use the old method of writing effects for custom buttons");
 define("_MI_formulize_USEOLDCUSTOMBUTTONEFFECTWRITINGDESC", "This should always be \"No\" unless this is an older installation that already has custom buttons that are dependent on the old method, which was based on the declaring human readable values, instead of the database values for elements.");
+
+define("_MI_formulize_FORMULIZELOGFILELOCATION", "Location to store Formulize log files");
+define("_MI_formulize_FORMULIZELOGFILELOCATIONDESC", "Formulize generates log files that contain the history of user actions, such as logging in and saving data. You can specify the full path to the folder where the log files are stored. You can turn off logging by specifying an empty or non-existent path.");
```

Comment:
Feels a little clunky that there isn't an explicit on/off for the logs. Especially since you're putting a default value which implies that logging is "enabled" but only so long as the directory exists.

Reply:
Add on/off preference